python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -323,6 +327,9 @@ def validate(self):
         if hasattr(self.getTemplate, "coaddName"):
             if self.getTemplate.coaddName != self.coaddName:
                 raise ValueError("Mis-matched coaddName and getTemplate.coaddName in the config.")
+        if self.doScaleDiffimVariance and self.doScaleTemplateVariance:
+            raise ValueError("If you are scaling the diffim variance, it is pointless to also "
+                             "scale the template variance. Please choose one or the other.")
```

Comment:
I would avoid error messages that judge the user. How about "Scaling the difference image variance and template variance are both set. Please choose one or the other."